edgexfoundry-holding/device-onvif-camera
# refactor: switch UpdateDevice calls to PatchDevice, and update buildDeviceName to use rfc3986

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [N/A] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

## Testing Instructions

- Build and run this version of the service.
- Ensure that devices are properly updated during checkstatus
  - this can be done by checking LastSeenTime and DeviceStatus fields

## New Dependency Instructions (If applicable)
